Source/Assets/PlayFabEditorExtensions/Editor/Scripts/Panels/PlayFabEditorAuthenticate.cs

```diff
@@ -66,9 +66,10 @@ public static void DrawAuthPanels()
             }
 
 
-            if (!string.IsNullOrEmpty(PlayFabEditorDataService.AccountDetails.email) && !isInitialized)
+            if (!string.IsNullOrEmpty(PlayFabEditorPrefsSO.Instance.DevAccountEmail) && !isInitialized)
             {
-                _userEmail = PlayFabEditorDataService.AccountDetails.email;
+                _userEmail = PlayFabEditorPrefsSO.Instance.DevAccountEmail;
+				PlayFabEditorPrefsSO.Save();
```

Comment:
>				 [](start = 0, length = 4)

Probably want to replace those tabs with spaces

Reply:
bah. Yes.

Reply:
Fixed.